src/pytorch_lightning/loggers/csv_logs.py

```diff
@@ -22,7 +22,7 @@
 import logging
 import os
 from argparse import Namespace
-from typing import Any, Dict, Optional, Union
+from typing import Any, Dict, Optional, Pathlike, Union
```

Comment:
added PathLike to account for situations when a user might pass in a Posix or Windows path